api/PaintRenderingContext2D.json

```diff
@@ -0,0 +1,2095 @@
+{
+  "api": {
+    "PaintRenderingContext2D": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaintRenderingContext2D",
+        "support": {
+          "chrome": {
+            "version_added": "65"
+          },
+          "chrome_android": {
+            "version_added": "65"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": "65"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "save": {
```

Comment:
I'm not really following the sort order of these features (doesn't seem to line up with the sorting implied by the spec or the Chrome IDL file). Maybe just sort these alphabetically?

Reply:
I've been asked not to add version numbers and sort in the same PR because it makes the additions hard to see.

Reply:
But this file (like all the files in the PR) is new, so it doesn't really make a difference for the diff. Personally, I'd prefer that these features have some discernable order before merging it

Reply:
You're right. Sorry I've got like 10 things going at a time.

Reply:
It's OK, completely understandable 👍